src/components/Questionnaire/ManageQuestionnaireTagsSheet.tsx

```diff
@@ -150,133 +151,136 @@ export default function ManageQuestionnaireTagsSheet({
+                  <Input
+                    id="tag-name"
+                    value={newTagName}
+                    onChange={(e) => setNewTagName(e.target.value)}
+                    placeholder={t("enter_tag_name")}
+                  />
                 </div>
-                <CareIcon
-                  icon={isCreateOpen ? "l-angle-up" : "l-angle-down"}
-                  className="h-4 w-4"
-                />
-              </Button>
-            </CollapsibleTrigger>
-            <CollapsibleContent className="mt-4 space-y-4">
-              <div className="space-y-2">
-                <Label htmlFor="tag-name">{t("tag_name")}</Label>
-                <Input
-                  id="tag-name"
-                  value={newTagName}
-                  onChange={(e) => setNewTagName(e.target.value)}
-                  placeholder={t("enter_tag_name")}
-                />
-              </div>
-              <div className="space-y-2">
-                <Label htmlFor="tag-slug">{t("tag_slug")}</Label>
-                <Input
-                  id="tag-slug"
-                  value={newTagSlug}
-                  onChange={(e) => setNewTagSlug(e.target.value)}
-                  placeholder={t("enter_tag_slug")}
-                />
-              </div>
-              <Button
-                onClick={handleCreateTag}
-                disabled={isCreating || !newTagName || !newTagSlug}
-                className="w-full"
-              >
-                {isCreating ? (
-                  <>
-                    <Loader2 className="mr-2 h-4 w-4 animate-spin" />
-                    {t("creating")}
-                  </>
-                ) : (
-                  t("create_tag")
-                )}
-              </Button>
-            </CollapsibleContent>
-          </Collapsible>
+                <div className="space-y-2">
+                  <Label htmlFor="tag-slug">{t("tag_slug")}</Label>
+                  <Input
+                    id="tag-slug"
+                    value={newTagSlug}
+                    onChange={(e) => setNewTagSlug(e.target.value)}
+                    placeholder={t("enter_tag_slug")}
+                  />
+                </div>
+                <Button
+                  onClick={handleCreateTag}
+                  disabled={isCreating || !newTagName || !newTagSlug}
+                  className="w-full"
+                >
+                  {isCreating ? (
+                    <>
+                      <Loader2 className="mr-2 h-4 w-4 animate-spin" />
+                      {t("creating")}
+                    </>
+                  ) : (
+                    t("create_tag")
+                  )}
+                </Button>
+              </CollapsibleContent>
+            </Collapsible>
+          </ScrollArea>
         </div>
 
-        <SheetFooter className="absolute bottom-0 left-0 right-0 p-4 border-t bg-background">
+        <SheetFooter className=" bottom-0 left-0 right-0 p-4 border-t bg-background">
```

Comment:
is this even a valid class? 🤔 

```suggestion
        <SheetFooter className="bottom-0 left-0 right-0 p-4 border-t">
```

Reply:
We don't have it defined. Removed from a few other places too